### PR TITLE
Portfolio Project Desktop Layout Pull Request

### PR DESCRIPTION
**Pull request by HafizMassamTabraizKhan**
Modify the index.html and added media queries in style.css for the desktop layout.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -321,3 +321,158 @@ form textarea {
...
 }
+
+.desktop-menu {
+  display: none;
+}
+
+/* ---- Desktop Version ---- */
+
+@media screen and (min-width: 768px) {
+  .menu-icon {
```
Comment (Favourezeugwa) [inline style.css:332]: - Great Job implementing the desktop version of your portfolio. Everything looks great on screen size 1024px and 1440px 👍🏽 .
- I'd like to kindly suggest that your work on the project cards sections on screen size 768px and that represents the view of your portfolio on tablets and iPad. Can we fix this? 
- Still on screen size 768px, there is some extra space on the right side of the portfolio. 

Here is what I mean 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| Your Project cards  |
| :---:   | 
|Screenshot 2023-03-30 at 11 15 52  |

|Extra space on the right |
| :---:   | 
|Screenshot 2023-03-30 at 11 16 10  |

⚠️ Kindly note that the extra space issue should be fixed on the headline section of your portfolio as well. 🤗

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @HafizMassamTabraizKhan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -321,3 +321,159 @@ form textarea {
...
```
Comment (levy002) [inline style.css:332]: - [OPTIONAL] Kindly make sure the style you added for maintaining your Portfolio responsiveness on the desktop version is not affecting the design of the mobile version. Kindly fix the mobile version design.

   ### YOUR MOBILE DESIGN

     screencapture-127-0-0-1-5500-index-html-2023-03-30-19_20_56 (1)

**Review by levy002 — APPROVED** [review]
## Hello @HafizMassamTabraizKhan 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to remove all comments and instructions in your README file in order to keep it clean and professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**